broker/src/main/java/org/apache/rocketmq/broker/processor/ReplyMessageProcessor.java

```diff
@@ -173,6 +173,7 @@ private PushReplyResult pushReplyMessage(final ChannelHandlerContext ctx,
         replyMessageRequestHeader.setProperties(requestHeader.getProperties());
         replyMessageRequestHeader.setReconsumeTimes(requestHeader.getReconsumeTimes());
         replyMessageRequestHeader.setUnitMode(requestHeader.isUnitMode());
+        replyMessageRequestHeader.setBname(requestHeader.getBname());
```

Comment:
`bname` is arcane and inconsistent with the existing field naming convention.  Why not use brokerName?